edge-hub/src/Microsoft.Azure.Devices.Routing.Core/endpoints/StoringAsyncEndpointExecutor.cs

```diff
@@ -1,153 +1,162 @@
-// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Is there some formatting issue here? Why are all lines showing up as diffs? Can you pls fix that.. hard to tell what has actually changed.